SpatialGDK/Documentation/content/authority.md

```diff
@@ -59,13 +59,15 @@ Actor on **non-owning client**:
 
 ## GDK authority
 
-As the GDK works with multiple [server-workers]({{urlRoot}}/content/glossary#worker), rather than a single server, authority needs to be dictated by SpatialOS so that authority is shared between server-workers. This means server-workers have authority over some Actors but don’t have authority over other Actors, depending on how SpatialOS assigns authority. **This is a key difference between the GDK and native Unreal networking!**
+Because you can use the GDK with multiple [server-workers]({{urlRoot}}/content/glossary#worker) using [zoning]({{urlRoot}}/content/glossary#zoning), authority needs to be dictated by SpatialOS so that it is shared between server-workers. This means server-workers have authority over some Actors but don’t have authority over other Actors, depending on how SpatialOS assigns authority. **This is a key difference between the GDK and native Unreal networking!**
```

Comment:
Thinking remove passive voice.
Remove "dictate" suggest allocate.
"share" could be confusing - server-worker instances never share authority as only one has authority at any time.
"have authority over some Actors but don’t have authority over other Actors" second half of sentence is redundant, I think?
Also they might only have authority over some Actors.

```suggestion
Because you can use the GDK with multiple [server-workers]({{urlRoot}}/content/glossary#worker) using [zoning]({{urlRoot}}/content/glossary#zoning), authority SpatialOS needs to specify authority, so that it can allocate authority appropriately between server-workers. This means that, when you use zoning, at any given moment, server-worker instances might have authority over only some Actors, depending on how SpatialOS assigns authority. **This is a key difference between the GDK and native Unreal networking!**
```

Reply:
This page needs some TLC. Will make these updates for now!